StartMetadata.go

```diff
@@ -25,13 +25,15 @@ import (
 	"strings"
 	"time"
 
+	logger "github.com/edgexfoundry/support-logging-client-go"
 	consulclient "github.com/edgexfoundry/consul-client-go"
 	notifications "github.com/edgexfoundry/support-notifications-client-go"
 )
 
 // DS : DataStore to retrieve data from database.
 var DS DataStore
 var notificationsClient = notifications.NotificationsClient{}
+var loggingClient = logger.NewClient(configuration.ApplicationName, "")
```

Comment:
Application Name gets read out of the configuration file below, see #6.

Thanks for the fixes!